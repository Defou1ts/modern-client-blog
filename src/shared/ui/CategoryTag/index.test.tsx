import React from 'react';

import { render, screen } from '@testing-library/react';

import renderer from 'react-test-renderer';

import { CategoryTag } from './index';

describe('CategoryTag', () => {
	it('renders a CategoryTag', () => {
		render(<CategoryTag tagName="Business" />);
		const text = screen.getByText('Business');
		expect(text).toBeInTheDocument();
		expect(text).toBeVisible();
	});

	it('CategoryTag snapshot', () => {
		const component = renderer.create(<CategoryTag tagName="Business" />);

		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
