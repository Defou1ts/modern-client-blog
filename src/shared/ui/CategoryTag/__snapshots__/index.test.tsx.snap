// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CategoryTag CategoryTag snapshot 1`] = `
<div
  className="tag"
>
  Business
</div>
`;
